docs/spec_draft.md

```diff
@@ -518,6 +519,40 @@ returned.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.cbrt
+
+### Semantics
+
+Performs element-wise cubic root operation on `operand` tensor and produces a
+`result` tensor, implementing the `rootn(x, 3)` operation from the IEEE-754
+specification.
```

Comment:
Thanks for the pointers! I strongly feel that we should drop this op.

At thin point we should add a ticket to track the drop this op from StableHLO and its addition to https://github.com/openxla/stablehlo/blob/main/stablehlo/dialect/ChloOps.td.

Reply:
"and close this PR". I think we should keep this PR because for better or for worse `cbrt` is already part of the StableHLO opset, so we'll need to keep it around for a while for compatibility reasons. At this point, let's limit ourselves to opening a ticket along the lines of "Consider moving CbrtOp to CHLO" and referencing this discussion there.

Reply:
I opened #586 to continue the discussion of potentially moving `cbrt` to CHLO.